src/oc_erchef/apps/oc_chef_wm/src/chef_wm_search.erl

```diff
@@ -118,7 +118,7 @@ handle_undefined_start(Start, _Req) ->
 decode(undefined) ->
     0;
 decode(Val) ->
-    list_to_integer(http_uri:decode(Val)).
```

Comment:
don't we want to use http_uri_decode here?

Reply:
@marcparadise Thought it was safe to remove it, which also simplifies the code.

Reply:
since we have the taken the implementation from OTP, it might be the best to leave the original calls to encode and decode as is?

Reply:
After discussion we agreed to leave it as it currently is.